### PR TITLE
Thunderbird profile lockdown

### PR DESCRIPTION
**Pull request by Xenopathic**
Same thing as #106, just for Thunderbird. Also removed the first run notices and splash screens for Firetray, Omnibar and Zotero.

Please review @Eldara98

### CODE REVIEW

File: linuxclientsetup/scripts/client-config [hunk 1]
```diff
@@ -332,6 +335,21 @@ if [[ $PROXYSERVER ]]; then
...
+		pref("network.proxy.ftp", "$PROXYSERVER.$DNSSUFFIX");
+		pref("network.proxy.ftp_port", 3128);
+		pref("network.proxy.socks", "$PROXYSERVER.$DNSSUFFIX");
+		pref("network.proxy.socks_port", 3128);
+		pref("network.proxy.gopher", "$PROXYSERVER.$DNSSUFFIX");
+		pref("network.proxy.gopher_port", 3128);
+		pref("network.proxy.type", 1);
+		pref("network.proxy.no_proxies_on", "localhost, 127.0.0.0/8, 192.168.0.0/16, 172.16.0.0/12, 10.0.0.0/8, $DNSSUFFIX, $PDC");
+	FOE
+OE
```
Comment (Eldara98) [inline linuxclientsetup/scripts/client-config]: This delimiter has no corresponding pair?

**Comment by Eldara98** [comment]
Generally LGTM, but I thought leaving the first run pages in was in the interest of improving UX? Seeing as the profiles are now uploaded, does it really matter if users see the first run pages?

File: linuxclientsetup/scripts/client-config (hunk 1 above)
Comment (Xenopathic) [inline linuxclientsetup/scripts/client-config]: Well spotted

**Comment by Xenopathic** [comment]
@Eldara98 The problem is, these are extensions that should be transparent to the user. A user shouldn't see the first run page for Firetray: they don't care about it. Likewise for Omnibar. I thought about leaving the Zotero first run page in, since that is the only relevant extension, but the way it does it is a bit obtrusive IMHO. I think the drop down notification 'Press here to open Zotero' is sufficient.

**Comment by Xenopathic** [comment]
@Eldara98 Please review that last commit. Thanks!

**Comment by Xenopathic** [comment]
OK, I've thoroughly tested the changes, and everything is working OK. Just waiting on your review @Eldara98 !

**Comment by Eldara98** [comment]
LGTM :+1:

**Pull request merged**